feature-utils/poly-analysis/src/analysis/root-analysis.js

```diff
@@ -0,0 +1,18 @@
+export default class RootAnalysis {
+    get id() {
+        return this.constructor.name;
+    }
+
+    get label() {
+        return RootAnalysis.Labels.TECH_DEMO;
+    }
+
+    get customReportData() {
+        return null;
+    }
+}
+
+RootAnalysis.Labels = Object.freeze({
+    NONE: null,
+    TECH_DEMO: "techDemo",
```

Comment:
This still relevant?

Reply:
We might need that again let's leave it in for now